libbeat/common/useragent/useragent.go

```diff
@@ -20,15 +20,29 @@ package useragent
 import (
 	"fmt"
 	"runtime"
+	"strings"
 
 	"github.com/elastic/beats/v7/libbeat/version"
 )
 
 // UserAgent takes the capitalized name of the current beat and returns
 // an RFC compliant user agent string for that beat.
-func UserAgent(beatNameCapitalized string) string {
-	return fmt.Sprintf("Elastic-%s/%s (%s; %s; %s; %s)",
-		beatNameCapitalized,
-		version.GetDefaultVersion(), runtime.GOOS, runtime.GOARCH,
-		version.Commit(), version.BuildTime())
+func UserAgent(beatNameCapitalized string, additional_agents ...string) string {
```

Comment:
```suggestion
func UserAgent(beatNameCapitalized string, additionalComments ...string) string {
```

Go uses camel case. And I think the user-agent RFC calls these "comments" IIRC.